frontend/server/src/Controllers/Course.php

```diff
@@ -3028,11 +3028,18 @@ public static function getStudentsInformationForSmarty(
             'problemTitles' => $problemTitles,
         ] = \OmegaUp\Cache::getFromCacheOrSet(
             \OmegaUp\Cache::SCHOOL_STUDENTS_PROGRESS,
-            "{$courseAlias}",
-            fn () => \OmegaUp\DAO\Courses::getStudentsInCourseWithProgressPerAssignment(
-                $course->course_id,
-                $course->group_id
-            ),
+            "$courseAlias",
```

Comment:
```suggestion
            $courseAlias,
```

aquí aún se puede evitar la cadenificación. Igual en L3092.